scripts/references/Dockerfile

```diff
@@ -1,21 +1,34 @@
-FROM r-base:4.0.0
+FROM rstudio/r-base:4.0-bullseye
 
+# Set working directory
 WORKDIR /usr/local/src/myscripts
 
+# Set environment variable for renv version
 ENV RENV_VERSION 0.14.0
 
-RUN rm /etc/apt/apt.conf.d/default
-RUN apt-get update -y
-RUN apt-get install -y dpkg-dev zlib1g-dev libssl-dev libffi-dev
-# procps is required for nextflow tower
-RUN apt-get install -y curl libcurl4-openssl-dev procps
-RUN R -e "install.packages('synapser', repos=c('http://ran.synapse.org', 'http://cran.fhcrc.org'))"
+# Update apt-get and install system dependencies (only install required)
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends \ 
+    dpkg-dev zlib1g-dev libssl-dev libffi-dev \
+    libcurl4-openssl-dev curl procps && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
-ENV PYTHON /usr/local/lib/R/site-library/PythonEmbedInR/bin/python3.6
+# Install R packages including remotes and renv
+RUN R -e "install.packages('remotes', repos = 'https://cloud.r-project.org')" && \
+    R -e "remotes::install_github('rstudio/renv', ref = '${RENV_VERSION}')" || true
 
-RUN R -e "install.packages('remotes', repos = c(CRAN = 'https://cloud.r-project.org'))"
-RUN R -e "remotes::install_github('rstudio/renv@${RENV_VERSION}')"
+# Install synapser with specific version
```

Comment:
I was just curious if there was a specific reason for sticking with that 0.11.7 and PythonEmbededInR instead of updating to the latest synapser? I'm wondering if there's something I should be aware of, like compatibility issues with the new version.

Reply:
No reason other than we already know the code works with this docker environment. Also because updating R env can be quite painful.